src/Resources/config/services.yml

```diff
@@ -9,9 +9,10 @@ services:
                 - ["setContainer", ["@service_container"]]
 
     contao.argument_resolver.model:
```

Comment:
You also have to adjust the name to `contao.model_argument_resolver` and move the definition to the right place (services are ordered alphabetically).